doc/release-notes.md

```diff
@@ -14,335 +14,215 @@ To receive notifications about updates, subscribe to the release mailing list:
 
   <https://sourceforge.net/projects/dogecoin/lists/dogecoin-releases>
 
-Releases are also announced on reddit:
-
-  <https://www.reddit.com/r/dogecoindev/>
 
 Compatibility
 ==============
 
-Dogecoin Core is extensively tested on Ubuntu Server LTS, Intel-based macOS
-and Windows 10.
+Dogecoin Core is extensively tested on Ubuntu Server LTS, macOS and Windows.
+Minimum OS compatibility can be found [in the INSTALL guide](../INSTALL.md).
+
 
-Dogecoin Core should also work on most other Unix-like systems but is not
-frequently tested on them.
 
 Notable changes
-===============
+================
 
```

Comment:
### Changes to RPC Authentication
#3352 The RPC authentication method has been updated and hardened to use cryptographically secure random numbers for generating passwords.